hail/python/hail/utils/struct.py

```diff
@@ -143,6 +143,29 @@ def select(self, *fields, **kwargs):
         -------
         :class:`.Struct`
             Struct containing specified existing fields and computed fields.
+
+        Examples
+        --------
+
+        Define a Struct 's'
+
+        >>> s = hl.Struct(foo=5, apple=10)
+
+        Keep just one original field
+
+        >>> s.select('foo')
+        hl.Struct(foo=5)
+
+        Add one new fields and keeps old field
```

Comment:
change "add one new fields" to "Add one new field"

change "keeps old field" to "keeps one old field"

Reply:
Fixed